Src/IdiomsUnitTest/ConstructorInitializedMemberAssertionTest.cs

```diff
@@ -475,6 +474,84 @@ public void VerifyTypeWithPublicReadOnlyFieldsIncorrectlyInitialisedViaConstruct
             // Teardown
         }
 
+        [Fact]
+        public void VerifyWellBehavedEnumFieldDoesNotThrow()
+        {
+            // Fixture setup
+            var dummyComposer = new Fixture();
+            var sut = new ConstructorInitializedMemberAssertion(dummyComposer);
+            var fieldInfo = typeof(ReadOnlyFieldInitializedViaConstructor<TestEnum>).GetField("Field");
+
+            // Exercise system and verify outcome
+            Assert.DoesNotThrow(() => sut.Verify(fieldInfo));
+            // Teardown
+        }
+
+        [Fact]
+        public void VerifyIllBehavedEnumFieldDoesThrow()
+        {
+            // Fixture setup
+            var dummyComposer = new Fixture();
+            var sut = new ConstructorInitializedMemberAssertion(dummyComposer);
+            var fieldInfo = typeof(ReadOnlyFieldIncorrectlyInitializedViaConstructor<TestEnum>).GetField("Field");
+
+            // Exercise system and verify outcome
+            Assert.Throws<ConstructorInitializedMemberException>(() => sut.Verify(fieldInfo));
+            // Teardown
+        }
+
+        [Fact]
+        public void VerifyWellBehavedEnumPropertyDoesThrow()
+        {
+            // Fixture setup
+            var dummyComposer = new Fixture();
+            var sut = new ConstructorInitializedMemberAssertion(dummyComposer);
+            var propertyInfo = typeof(ReadOnlyPropertyInitializedViaConstructor<TestEnum>).GetProperty("Property");
+
+            // Exercise system and verify outcome
+            Assert.DoesNotThrow(() => sut.Verify(propertyInfo));
+            // Teardown
+        }
+
+        [Fact]
+        public void VerifyIllBehavedEnumPropertyDoesThrow()
+        {
+            // Fixture setup
+            var dummyComposer = new Fixture();
+            var sut = new ConstructorInitializedMemberAssertion(dummyComposer);
+            var propertyInfo = typeof(ReadOnlyPropertyIncorrectlyInitializedViaConstructor<TestEnum>).GetProperty("Property");
+
+            // Exercise system and verify outcome
+            Assert.Throws<ConstructorInitializedMemberException>(() => sut.Verify(propertyInfo));
+            // Teardown
+        }
+
+        [Fact]
+        public void VerifyDefaultOnlyEnumFieldDoesThrow()
+        {
+            // Fixture setup
+            var dummyComposer = new Fixture();
+            var sut = new ConstructorInitializedMemberAssertion(dummyComposer);
+            var fieldInfo = typeof(ReadOnlyFieldIncorrectlyInitializedViaConstructor<TestDefaultOnlyEnum>).GetField("Field");
+
+            // Exercise system and verify outcome
+            Assert.Throws<ConstructorInitializedMemberException>(() => sut.Verify(fieldInfo));
+            // Teardown
+        }
+
+        [Fact]
+        public void VerifyDefaultOnlyEnumPropertyDoesThrow()
+        {
+            // Fixture setup
+            var dummyComposer = new Fixture();
```

Comment:
To be honest, I was following the convention laid out by existing unit tests in this source file.  

Thinking about it, "dummy" might suggest a [dummy object](http://xunitpatterns.com/Dummy%20Object.html) which is not used by the test.  In this case, it is a critical stand-in for `ISpecimenBuilder`.

Any objections to `fixture` instead?   Shall I change all pre-existing instances as well? 
I assume we frown on inconsistency, but so to is mucking around too far outside the scope of a PR.